internal/providers/pluginfw/products/alert_v2/data_alert_v2.go

```diff
@@ -154,21 +155,21 @@ func (m AlertV2Data) Type(ctx context.Context) attr.Type {
 
 func (m AlertV2Data) ApplySchemaCustomizations(attrs map[string]tfschema.AttributeBuilder) map[string]tfschema.AttributeBuilder {
 	attrs["create_time"] = attrs["create_time"].SetComputed()
-	attrs["custom_description"] = attrs["custom_description"].SetOptional()
-	attrs["custom_summary"] = attrs["custom_summary"].SetOptional()
-	attrs["display_name"] = attrs["display_name"].SetOptional()
+	attrs["custom_description"] = attrs["custom_description"].SetComputed()
+	attrs["custom_summary"] = attrs["custom_summary"].SetComputed()
+	attrs["display_name"] = attrs["display_name"].SetComputed()
 	attrs["effective_run_as"] = attrs["effective_run_as"].SetComputed()
```

Comment:
For a lot of data sources, SetOptional and SetRequires is being updated to SetComputed, is this expected?